public/static/locales/eng/privacy-policy.md

```diff
@@ -2,108 +2,98 @@
-
-• raddupptökum
-
-• lýðfræðilegum upplýsingum s.s. aldur, kyn og móðurmál.
-
-• samskiptaupplýsingum, s.s. tölvupósti
-
-• stafrænum fótsporum, s.s. nethegðun
-
-• upplýsingar tengdar samþykkis forsjáraðila fyrir barn s.s. kennitala barns og netfang forsjáraðila
+RU emphasizes the processing of only the personal information that is necessary in accordance with the purpose behind the collection of the information. RU does not further process personal information for unrelated purposes unless an individual is notified of this and the authorization on which the processing is based. RU collects, as applicable at any given time, the following personal information:
 
-• undir hvaða formerkjum þú, lest s.s. nafn skóla
+-   Voice recordings
+-   Demographic information e.g. age, gender and mother tongue.
+-   Contact information, if you create an account then we store your email.
+-   Digital cookies, e.g. online behaviour.
+-   Information related to the consent of custodians for the children participating, e.g. child's ID number and email address of the guardian.
+-   For which school you are reading for during the Samrómur reading competition.
 
-HR vinnur persónuupplýsingar einkum í þeim tilgangi að:
+RU processes personal information primarily for the purpose of:
 
-• búa til opið íslenskt raddgagnasafn til að styðja við og efla rannsóknir og þróun máltæknilausna
+-   creating an open Icelandic voice database to support and strengthen research and development of language technology solutions in Icelandic.
 
-Þegar þú notar vefinn [samrómur.is](https://samromur.is/) söfnum við upplýsingum um notkun þína, þ.e. IP tölu, tegund eða útgáfu vafra sem þú notar, tímasetningu og tímalengd heimsóknar og hvaða undirsíður þú heimsækir innan heimasíðu Samróms. Hér má finna nánari upplýsingar um notkun okkar á vafrakökum (e. cookies), [samrómur.is/vafrakökustefna](https://www.samromur.is/vafrakokustefna).
+When you use the website Samromur.is, we collect information about your use, i.e. IP address, type or version of browser you use, timing and duration of the visit and which subpages you visit within Samrómur website. Here you can find more information about our use of [cookies](https://www.samromur.is/vafrakokustefna).
 
-# Lagalegur grundvöllur fyrir vinnslu
+# Legal basis for processing
 
-HR safnar og vinnur persónuupplýsingar byggt á heimild í 6. tl. 1. mgr. 9. gr. persónuverndarlaga, þ.e. vegna lögmætra hagsmuna stofnunarinnar. Eftir atvikum kann HR einnig að vinna persónuupplýsingar byggt á samþykki þínu, sbr. 1. tl.1. mgr. 9. gr. persónuverndarlaga, s.s. ef þú skráir þig á póstlista okkar.
+RU collects and processes personal information based on the authority in item 6. Paragraph 1 Article 9 the Privacy Act, i.e. due to the legitimate interests of RU. Depending on the circumstances, RU may also process personal information based on your consent, cf. 1. tl.1. mgr. Article 9 the Privacy Act, e.g. if you subscribe to our mailing list.
```

Comment:
it says t1.1. mgr. here. i think those shouldn't be here since it's for the icelandic version i believe

Reply:
You mean remove it? Or Isn't it referring to the same article?

Reply:
what does cf. 1. t1.1. mgr. mean in english? what does cf stand for? what does the t stand for? what does mgr stand for? in english that is

Reply:
updated it to reflect "according to item 1 in paragraph 1 in article 9 of the Privacy act